jest.config.ci.js

```diff
@@ -15,9 +15,6 @@ module.exports = {
       'jest-junit',
       {outputDirectory: 'reports/junit', outputName: 'js-test-results.xml'},
     ],
-    [
-      'jest-silent-reporter',
-      {showPaths: true, showWarnings: true, useDots: true},
-    ],
+    'default',
```

Comment:
😢

Reply:
heh, soz. At the time it was useful to get test timings and such, now that we're down to actual test failures it should be good to move back 😉